### PR TITLE
Set "grdtnFlg" to "Y" once final submission is approved

### PR DESCRIPTION
**Pull request by ajkiessl**

### CODE REVIEW

File: spec/models/lionpath/lionpath_export_payload_spec.rb [hunk 1]
```diff
@@ -74,7 +75,7 @@
 
       it 'sets grdtnFlg to Y' do
```
Comment (jlandiseigsti) [inline spec/models/lionpath/lionpath_export_payload_spec.rb:76]: Should this test be renamed?

**Review by jlandiseigsti — CHANGES_REQUESTED** [review]
Just one quick change

File: spec/models/lionpath/lionpath_export_payload_spec.rb (hunk 1 above)
Comment (ajkiessl) [reply spec/models/lionpath/lionpath_export_payload_spec.rb:76]: Ah yes!

**Review by jlandiseigsti — APPROVED** [review]
👍

**Pull request merged**